$DragonFly: src/contrib/top/patches/Attic/commands.c.patch,v 1.1 2004/05/29 05:11:15 hmp Exp $
Index: contrib/top/commands.c
===================================================================
RCS file: /cvs/DragonFly/src/contrib/top/commands.c,v
retrieving revision 1.2
diff -u -p -u -r1.2 commands.c
--- contrib/top/commands.c	17 Jun 2003 04:24:07 -0000	1.2
+++ contrib/top/commands.c	29 May 2004 01:14:21 -0000
@@ -73,11 +73,13 @@ d       - change number of displays to s
 e       - list errors generated by last \"kill\" or \"renice\" command\n\
 i       - toggle the displaying of idle processes\n\
 I       - same as 'i'\n\
+T	- toggle the displaying of threads\n\
+S	- toggle the displaying of system processes\n\
 k       - kill processes; send a signal to a list of processes\n\
 n or #  - change number of processes to display\n", stdout);
 #ifdef ORDER
 	fputs("\
-o       - specify sort order (pri, size, res, cpu, time)\n", stdout);
+o       - specify sort order (pri, size, res, cpu, time, thr)\n", stdout);
 #endif
 	fputs("\
 r       - renice a process\n\
